test/apify/actor.test.ts

```diff
@@ -991,44 +991,36 @@ describe('Actor', () => {
         test('should work', async () => {
             const record = { foo: 'bar' };
             const defaultStore = await KeyValueStore.open();
+            const setValueSpy = jest.spyOn(defaultStore, 'setValue');
 
-            const oldSet = defaultStore.setValue;
-            defaultStore.setValue = async (key, value) => {
-                expect(key).toBe('key-1');
-                expect(value).toBe(record);
-            };
+            setValueSpy.mockImplementation(async (key, value) => {
+                expect(key).toEqual('key-1');
+                expect(value).toEqual(record);
```

Comment:
those assertions will result in promise rejection instead of test failure, that's another part of the problem in that issue